src/Database/PostgreSQL/Consumers/Config.hs

```diff
@@ -21,12 +22,16 @@ data Action
   | RerunAfter Interval
   | RerunAt UTCTime
   | Remove
-    deriving (Eq, Ord, Show)
+  deriving (Eq, Ord, Show)
 
 -- | Result of processing a job.
 data Result = Ok Action | Failed Action
   deriving (Eq, Ord, Show)
 
+-- | The mode the consumer will run in.
+data Mode = Standard | Duplicating (RawSQL ())
```

Comment:
`Duplicating` should probably be a non-empty array of expressions as one may want to be able to de-duplicate on more than one expression. And the SQL expression type should be just `SQL` and not `RawSQL ()`.

Reply:
`RawSQL ()` is fine, it's for "sql literals", i.e. values that can't hold parameters.